### PR TITLE
  publish CallerDepthFromContext, this is needed for creating another adapter

### PR DESCRIPTION
**Pull request by podhmo**
yes.

### CODE REVIEW

File: context.go [hunk 1]
```diff
@@ -55,7 +55,8 @@ func adapterFromContext(ctx context.Context) Adapter {
 	return lc
 }
 
-func callDepthFromContext(ctx context.Context) int {
+// CallerDepthFromContext return callerDepth int value, this value is used by finding caller position, usually dones't have to remember it.
```
Comment (tomono-wacul) [inline context.go]: typo: dones't

Comment (podhmo) [reply context.go]: oops

Comment (podhmo) [reply context.go]: fixed

**Comment by tomono-wacul** [comment]
travis failed.
```
./adapter.go:59:9: undefined: callDepthFromContext
./context.go:69:46: undefined: callDepthFromContext
```

**Review by tomono-wacul — APPROVED** [review]
lgtm 🙋‍♂

**Pull request merged**